ISISNeutronMuon/MDANSE
# Trajectory tab and 3D view updates.

**Description of work**
- Improved the trajectory loading performance. 42,592 atoms can be loaded in a few seconds.
- Added bonding to 3D view.
- Added unit cell to 3D view.
- Added method for faster access to atom database properties.
- Updated trajectory panel so that they are not reloaded on trajectory selection change.
- Optimized the build connectivity in MDANSE graph.

![Animation12](https://github.com/ISISNeutronMuon/MDANSE/assets/14276033/44be42b3-9ffa-4e4c-9d61-ee1066342777)

**To test**
Load a few trajectories up in the trajectory tab and check 3D view is working correctly.
